src/main/java/org/popcraft/chunky/command/ContinueCommand.java

```diff
@@ -13,6 +13,10 @@ public ContinueCommand(Chunky chunky) {
     }
 
     public void execute(CommandSender sender, String[] args) {
+        if (chunky.getConfigStorage().loadTasks().size() == 0) {
+            sender.sendMessage(chunky.message("format_continue_no_tasks", chunky.message("prefix")));
+            return;
+        }
         final Map<World, GenerationTask> generationTasks = chunky.getGenerationTasks();
         chunky.getConfigStorage().loadTasks().forEach(generationTask -> {
```

Comment:
You should probably only call loadTasks once since this is going to read from file.